modules/api/src/main/java/org/apache/fluo/api/data/Span.java

```diff
@@ -194,6 +194,11 @@ public String toString() {
         + ((endInclusive && !end.equals(RowColumn.EMPTY)) ? "]" : ")");
   }
 
+  @Override
+  public int hashCode() {
+    return Objects.hash(start, end, startInclusive, endInclusive);
```

Comment:
This method is also new to me.  We can use these methods in other places.  Opened #740